content/en/user-guide/security-testing/live-policy-stream/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Live Policy Stream"
+linkTitle: "Live Policy Stream"
+weight: 3
+description: Generate a stream of IAM policies as requests are coming into LocalStack using Live Policy Stream.
+---
+
+## Introduction
+
+The Live Policy Policy Stream generates a steady stream of policies along with their corresponding principals or resources. When a request is made, it initially shows the principal or resource to which the policy is to be connected. This is typically a service resource for resource-based policies, or an IAM principal for other cases. Subsequently, it displays the suggested policy. This feature aids in identifying the correct permissions for cloud applications and can help spot logical errors, such as unexpected actions in a policy.
+
+## Getting started
+
+This guide is designed for users who are new to the Live Policy Stream. It assumes you have basic knowledge of the AWS CLI (and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script).
+
+### Start your LocalStack container
+
+To experiment with the IAM Policy Stream, initiate LocalStack using these flags:
+
+1.  Enable debugging: `DEBUG=1`
+2.  Set your LocalStack API key: `LOCALSTACK_API_KEY=<Your_API_Key>`
+3.  Activate IAM enforcement: `ENFORCE_IAM=1`
+
+To utilize the IAM Policy Stream without enforcing IAM policies, set `IAM_SOFT_MODE=1`. You can execute the following command in your terminal to start your LocalStack container:
+
+{{< command >}}
+$ DEBUG=1 ENFORCE_IAM=1 IAM_SOFT_MODE=1 localstack start
```

Comment:
```suggestion
$ DEBUG=1 IAM_SOFT_MODE=1 localstack start
```
I removed the requirements for both options being set. Now IAM_SOFT_MODE alone suffices (if you set both, it will behave the same as if only IAM_SOFT_MODE is set, so backwards compatible)